kontra.js

```diff
@@ -3357,6 +3381,192 @@ class Quadtree {
 
 var Quadtree$1 = Factory(Quadtree);
 
+class Scene {
+
+  /**
+   * A scene object for organizing a group of objects that will update and render together.
+   *
+   * ```js
+   * import { Scene, Sprite } from 'kontra';
+   *
+   * sprite = Sprite({
+   *   x: 100,
+   *   y: 200,
+   *   width: 20,
+   *   height: 40,
+   *   color: 'red'
+   * });
+   *
+   * scene = Scene({
+   *   id: 'game',
+   *   children: [sprite]
+   * });
+   *
+   * scene.render();
+   * ```js
+   *
+   * @class Scene
+   *
+   * @param {Object} properties - Properties of the scene.
+   * @param {String} properties.id - The id of the scene.
+   * @param {String} [properties.name=properties.id] - The name of the scene. Used by screen readers to identify each scene. Use this property to give the scene a human friendly name. Defaults to the id.
+   * @param {Object[]} [properties.children=[]] - The children of the scene.
+   * @param {...*} properties.props - Any additional properties you need added to the scene. For example, if you pass `Scene({counter: 0})` then the scene will also have a property of the same name and value. You can pass as many additional properties as you want.
+   */
+  constructor(properties) {
+    /**
+     * The id of the scene.
+     * @memberof Scene
+     * @property {String} id
+     */
+
+    /**
+     * The name of the scene. Used by screen readers to identify each scene. Use this property to give the scene a human friendly name.
+     * @memberof Scene
+     * @property {String} name
+     */
+    this.name = properties.id;
+
+    // add all properties to the scene, overriding any defaults
+    Object.assign(this, properties);
+
+    /**
+     * The scenes children objects.
+     * @memberof Scene
+     * @property {Object[]} children
+     */
+    this.children = [];
+
+    // create an accessible DOM node for screen readers
+    // dn = dom node
+    const section = this._dn = document.createElement('section');
```

Comment:
How does this node help users depending on screen readers without optimizing the whole application for screen readers?

Reply:
The node is primarily for screen readers and keyboard users. I [wanted accessibility baked into the framework](https://github.com/straker/kontra/issues/124) if it started adding UI elements.